File: OOP/Week 07/Task/item.py
class Item:

    def __init__(self, title, playing_time):
        self.title = title
        self.playing_time = playing_time
        self.got_it = False
        self.comment = ''

    def set_comment(self, comment):
        self.comment = comment

    def get_comment(self):
        return self.comment
    
    def set_owned(self, owned):
        self.got_it = owned

    def get_owned(self):
        return self.got_it

    def info(self):
        print("Title:", self.title)
        print("Playing time:", self.playing_time)
        print("Comment:", self.comment)
        print("Owned:", self.got_it)

File: OOP/Java OOP/Week 09/Task02/Item.java
package Task02;

public class Item {
    protected String title;
    protected String playingTime;
    protected String comment;
    protected boolean gotIt;

    Item(String title, String playingTime) {
        this.title = title;
        this.playingTime = playingTime;
        this.comment = "";
        this.gotIt = false;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }

    public String getComment() {
        return this.comment;
    }

    public void setOwned(boolean owned) {
        this.gotIt = owned;
    }

    public boolean getOwned() {
        return this.gotIt;
    }

    public void info() {
        System.out.println("Title: " + this.title);
        System.out.println("Playing time: " + this.playingTime);
        System.out.println("Comment: " + this.comment);
        System.out.println("Owned: " + this.gotIt);
    }
}
